src/test/resources/projects/maven/jakarta-sample/src/main/java/io/openliberty/sample/jakarta/annotations/IncorrectPostConstructAnnotation.java

```diff
@@ -0,0 +1,19 @@
+package io.openliberty.sample.jakarta.annotations;
+
+import jakarta.annotation.Resource;
+import random.test.pkg.on.PostConstruct;
+
+@Resource(type = Object.class, name = "aa")
+public class IncorrectPostConstructAnnotation {
+
+    @PostConstruct
```

Comment:
@dessina-devasia , just to confirm , I can see that some of the annotation is having parenthesis () , that is @Generated() above , but for this case there is no parenthesis. Can we use both?